tasks/main.yml

```diff
@@ -253,7 +253,7 @@
     - name: Set the worker specific tftp files
       template:
         src: ../templates/pxe-worker.j2
-        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | regex_replace (':', '-')}}"
+        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | lower | regex_replace (':', '-')}}"
```

Comment:
Same as the masters section...

Doesn't the `lower` in the `with_items` section satisfy the requirement?

Reply:
You are right. Fixed